
This patch obviates the requirement in the distributed code to download a copy of
ODPI-C for direct compilation into the module.

This patch has been suggested upstream, via 
https://github.com/oracle/python-cx_Oracle/issues/103

--- cx_Oracle-6.1/src/cx_Oracle.c.orig	2017-10-30 14:32:53.530420385 +1000
+++ cx_Oracle-6.1/src/cx_Oracle.c	2017-10-30 14:34:15.644978118 +1000
@@ -725,10 +725,11 @@
     Module_Initialize();
 }
 #endif
 
 
+#if defined(ODPI_NOT_PRESENT)
 //-----------------------------------------------------------------------------
 // include all DPI files
 //-----------------------------------------------------------------------------
 
 #include "dpiConn.c"
@@ -751,6 +752,6 @@
 #include "dpiRowid.c"
 #include "dpiStmt.c"
 #include "dpiSubscr.c"
 #include "dpiUtils.c"
 #include "dpiVar.c"
-
+#endif
